packages/@orchidui-vue/src/Charts/PieChart/OcPieChart.vue

```diff
@@ -1,52 +1,61 @@
 <template>
-  <div class="flex flex-col items-center">
-    <div ref="pieChart" class="h-full w-full" />
+  <div>
+    <div class="flex flex-col items-center relative h-full w-full">
+      <div ref="pieChart" class="h-full w-full" />
 
-    <!--  Here we are using custom legend with toggleLegendName(name) for saving logic of buttons  -->
-    <div v-if="showLegend" class="flex gap-x-5">
-      <Tooltip
-        v-for="item in options.series[0].data"
-        :key="item.name"
-        position="top"
-        class="flex items-center"
-        :distance="10"
-        @click="toggleLegendName(item.name)"
-      >
-        <template #default>
-          <div
-            class="flex items-center gap-x-2 cursor-pointer"
-            :class="!legendSelected[item.name] && 'grayscale'"
-          >
+      <!--  Here we are using custom legend with toggleLegendName(name) for saving logic of buttons  -->
+      <div v-if="showLegend" class="flex gap-x-5">
+        <Tooltip
+          v-for="item in options.series[0].data"
+          :key="item.name"
+          position="top"
+          class="flex items-center"
+          :distance="10"
+          @click="toggleLegendName(item.name)"
+        >
+          <template #default>
             <div
-              class="w-3 h-3 rounded-full"
-              :style="{ background: item.itemStyle.color }"
-            />
-            <img v-if="item.name" :src="legendImages[item.name]" />
-            <span v-else>Other</span>
-          </div>
-        </template>
-        <template #popper>
-          <div
-            class="py-2 text-sm text-oc-text-400 font-medium px-3 max-w-[217px]"
-          >
-            {{ legendTooltipText[item.name] }}
-          </div>
-        </template>
-      </Tooltip>
+              class="flex items-center gap-x-2 cursor-pointer"
+              :class="!legendSelected[item.name] && 'grayscale'"
+            >
+              <div
+                class="w-3 h-3 rounded-full"
+                :style="{ background: item.itemStyle.color }"
+              />
+              <img v-if="item.name" :src="legendImages[item.name]" />
+              <span v-else>Other</span>
+            </div>
+          </template>
+          <template #popper>
+            <div
+              class="py-2 text-sm text-oc-text-400 font-medium px-3 max-w-[217px]"
+            >
+              {{ legendTooltipText[item.name] }}
+            </div>
+          </template>
+        </Tooltip>
+      </div>
     </div>
+    <dummy-data v-if="dummyData" absolute>
+      <slot name="dummy-data-description">
+        Demo reports will be replaced once <br />
+        you made transactions
```

Comment:
this text like in dummy-data component, you can remove it and only register component, if customize you can use v-if="hasDummySlots" <script setup>
import { useSlots } from 'vue'
const slots = useSlots()
</script>
example 
<template>
  <div v-if="slots.content" class="classname">
    <slot name="content"></slot>
  </div>
</template>